packages/cfd/src/Containers/cfd-top-up-demo-modal.tsx

```diff
@@ -3,45 +3,28 @@ import SuccessDialog from '../Components/success-dialog.jsx';
 import { Icon, Modal, Button, Money, Text } from '@deriv/components';
 import { getCFDPlatformLabel } from '@deriv/shared';
 import { localize, Localize } from '@deriv/translations';
-import { DetailsOfEachMT5Loginid } from '@deriv/api-types';
-import RootStore from '../Stores/index';
-import { connect } from '../Stores/connect';
-import { TDxCompanies, TMtCompanies } from '../Stores/Modules/CFD/Helpers/cfd-config';
+import { TMtCompanies } from '../Stores/Modules/CFD/Helpers/cfd-config';
 import { getTopUpConfig } from '../Helpers/constants';
-
-type TExtendedCurrentAccount = DetailsOfEachMT5Loginid & {
-    display_login: string;
-    category: string;
-    type: string;
-};
+import { observer, useStore } from '@deriv/stores';
+import { useCfdStore } from '../Stores/Modules/CFD/Helpers/useCfdStores';
```

Comment:
We can have an alias for this import since it is used in almost all of our components and we can have shorter imports.

Reply:
Same reason for this as well, check the above comment @hirad-deriv